opencti-platform/opencti-graphql/src/domain/file.js

```diff
@@ -46,16 +47,16 @@ export const filesMetrics = async (context, user) => {
   const potentialIndexingFiles = await allFilesForPaths(context, user, fileOptions.paths, fileOptions.opts);
   const remainingFilesCount = await allRemainingFilesCount(context, user, fileOptions.paths, fileOptions.opts);
   const metricsByMimeType = [];
-  const filesByMimetype = R.groupBy((f) => f.metaData.mimetype, potentialIndexingFiles);
-  const mimeTypesGroups = Object.keys(filesByMimetype);
-  for (let index = 0; index < mimeTypesGroups.length; index += 1) {
-    const mimeType = mimeTypesGroups[index];
-    metricsByMimeType.push({
-      mimeType,
-      count: filesByMimetype[mimeType].length,
-      size: R.sum(filesByMimetype[mimeType].map((file) => file.size)),
-    });
-  }
+  supportedMimeTypes.forEach((mimeType) => {
+    const mimeTypeFiles = potentialIndexingFiles.filter((file) => file.metaData.mimetype?.startsWith(mimeType));
+    if (mimeTypeFiles.length > 0) {
+      metricsByMimeType.push({
+        mimeType,
+        count: mimeTypeFiles.length,
+        size: R.sum(mimeTypeFiles.map((file) => file.size)),
```

Comment:
If you want to remove Ramda.sum you can use reduce instead :
`
mimeTypeFiles.map((file) => file.size).reduce((accumulator, currentValue) => accumulator + currentValue, 0)
`

Reply:
Oh thanks for the suggestion, I find it however more readable to use R.sum than the reduce. Usually I avoid using Ramda, but there are a few cases where the functions are much easier to read : R.sum and R.groupBy